src/components/vm/disks/vmDiskColumns.jsx

```diff
@@ -39,6 +39,7 @@ const _ = cockpit.gettext;
 
 export const DISK_SOURCE_LIST = [
     { name: "file", label: _("File") },
+    { name: "dev", label: _("Block device") },
```

Comment:
The commit message says "type block", but this says "dev"...  I guess that's what the XML fragment is trying to explain?

Reply:
Yes, exactly. This is how the code works in general.  A disk XML might look like this:

```
<disk type='file' device='disk'>
      <driver name='qemu' type='qcow2' discard='unmap'/>
      <source file='/var/lib/libvirt/images/foo.qcow2'/>
      <target dev='vda' bus='virtio'/>
      <address type='pci' domain='0x0000' bus='0x04' slot='0x00' function='0x0'/>
</disk>
```

Cockpit ignores the `type` attribute of `<disk>` and only looks at the `<source>` element. If that has a `file` attribute, for example, the "Source" column in the table gets a entry with "File  /var/lib/libvirt/images/foo.qcow2".

This PR extends this to also recognize `<source>` elements with `dev` attributes. They only happen with `<disk>` elements with `type="block"`.  I hope. :)